### ISSUE TITLE
Dataset comments

### ISSUE THREAD

**Issue by Kjos** [legolas123/cv-tricks.com#7]
Hi,
Thanks for putting this up on github. It got me started with tensorflow.
However some comments;
I think the dataset shows a lot of underfitting. I get a validation score of 0.5 at end, or even below.
I connected tensorboard and the learning graph shows classic underfitting  signs (straight learning rate).
Perhaps it doesn't help that the images have wide ranging aspect ratios?
I tried to insert horizontal flipped images as well, but it doesn't help that much.

**Comment by sardarwasif** [comment]
Hello,
I am training neural network using this code (cv-tricks.com-master\cv-tricks.com-master\Tensorflow-tutorials\tutorial-2-image-classifier\train) on my own data set of road traffic signs in which i have left and right road signs. I have changed the directory as follows:

classes = os.listdir("D:\My_tensorflow\data_set\Train")
train_path="D:\My_tensorflow\data_set\Train"
test_path="D:\My_tensorflow\data_set\Test"

when i run this code it gives this kind of out put and error:
Number of files in Training-set:    0
Number of files in Validation-set: 0
error is:
assert batch_size <= self._num_examples

Kindly help me to figure out this problem.
Thanks
